make/autoconf/flags-cflags.m4

```diff
@@ -384,7 +384,7 @@ AC_DEFUN([FLAGS_SETUP_OPTIMIZATION],
     C_O_FLAG_DEBUG="-Od"
     C_O_FLAG_DEBUG_JVM=""
     C_O_FLAG_NONE="-Od"
-    C_O_FLAG_SIZE="-Os"
+    C_O_FLAG_SIZE="-O1"
```

Comment:
This appears to be from 9b1db7246ab6518e66abd3688bbd9107394f5d75 which is not included in 24.0.1+9.